Readme.md

```diff
@@ -153,6 +153,15 @@ other, even if they do not modify the same records, e.g. if a commit only update
 subject will ensure the nested records are synced when the record is synced even if the record and its nested records
 are never modified together in the same commit.
 
+The `:except` option can be passed to `stage_changes` to exclude certain tables from a commit. This is useful if
+inconsequential updates to a certain table (e.g. counter cache updates) are causing records to be linked together unnecessarily.
```

Comment:
Should note that the commit entries are still created, just not a part of the commit.